packages/gatsby-plugin-utils/src/reporter.ts

```diff
@@ -0,0 +1,16 @@
+import c from "ansi-colors"
+// We don't want to depend on the whole of gatsby-cli, so we can't use reporter
+export const reporter = {
+  info: (message: string): void => console.log(message),
+  verbose: (message: string): void => console.log(message),
+  log: (message: string): void => console.log(message),
+  success: (message: string): void =>
+    console.log(c.green(c.symbols.check + ` `) + message),
+  error: (message: any): void =>
+    console.error(c.red(c.symbols.cross + ` `) + message),
```

Comment:
```suggestion
    console.error(c.red(c.symbols.cross + ` `), message),
```

If this is `any`, then you'll need to change this